ci/cloudbuild/builds/lib/integration.sh

```diff
@@ -163,6 +163,8 @@ function integration::bazel_with_emulators() {
     "google/cloud/pubsublite/..."
     # Unified Rest Credentials test
     "google/cloud:internal_unified_rest_credentials_integration_test"
+    # The OpenTelemetry integration tests are limited by quota
+    "google/cloud/opentelemetry/integration_tests/..."
```

Comment:
I think this is the opposite of what you want. Note that they are already in this list on L161.

I think we want to remove them from this list so they don't run.

Reply:
We will address in a follow up.